include/ignition/gui/PlottingInterface.hh

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_GUI_PLOTTINGINTERFACE_HH_
+#define IGNITION_GUI_PLOTTINGINTERFACE_HH_
+
+#include <QObject>
+#include <QTimer>
+#include <QString>
+#include <QVariant>
+#include <bits/stdc++.h>
+#include <google/protobuf/message.h>
+#include <google/protobuf/descriptor.h>
+#include <ignition/transport/Node.hh>
+#include <ignition/transport/MessageInfo.hh>
+#include <ignition/transport/Publisher.hh>
+#include <map>
+#include <set>
+#include <string>
+#include <memory>
```

Comment:
alphabetize, and need to follow order of headers https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes 

I'd recommend following this rule for all other headers in files to avoid codecheck warnings on CI.